src/themes/default/components/core/blocks/SearchPanel/SearchPanel.vue

```diff
@@ -65,11 +73,46 @@
 <script>
 import SearchPanel from '@vue-storefront/core/compatibility/components/blocks/SearchPanel/SearchPanel'
 import ProductTile from 'theme/components/core/ProductTile'
+import CategoryPanel from 'theme/components/core/blocks/Category/CategoryPanel'
 import VueOfflineMixin from 'vue-offline/mixin'
 
 export default {
   components: {
-    ProductTile
+    ProductTile,
+    CategoryPanel
+  },
+  computed: {
+    categories () {
+      return this.products.map(product => {
+        return product.category
+      })
+    },
+    selectedCategory () {
```

Comment:
this should be in this component data section binded to category-panel by v-model